stories/cellsContainer/Wrapper.svelte

```diff
@@ -0,0 +1,70 @@
+<script>
+  import CellsContainer from '../../src/components/CellsContainer/CellsContainer.svelte';
+
+  // export let count = 0;
+
+  let tooltipText = 'Tooltip Text';
+  let color = '#6495ED';
+  let hoverColor = '#FF7F50';
+  let size = 20;
+  let count = 30;
+  let gap = 2;
+  $: getAdditionalRectProps = (i) => ({ tooltipText });
+
+  const inputClass =
+    'border-2 focus:shadow-outline duration-200 border-gray-600 px-4 py-1 rounded';
+  const inputNumberClass = `${inputClass} w-24 pr-0`;
+  const hClass = 'text-center text-lg font-bold mt-4';
+  const containerClass = 'flex items-center justify-center mt-2';
+</script>
+
+<h2 class={hClass}>Size</h2>
```

Comment:
Looks like a duplication, maybe it would make sense to have a separate component for that?